functions/go/set-default-labels/README.md

```diff
@@ -0,0 +1,62 @@
+# set-default-labels
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `set-default-labels` function applies KPT label convention to your KPT package.
+
+
+### `app.kubernetes.io/name` 
+
+Running the function can add this label using the value from the root `Kptfile` name.
+
+This convention believes that all KRM resources under the same kpt package should be served for a specific application.
+An application (or KPT package) can be composed by other applications (nested sub KPT packages). 
+So the function can accept multiple Kptfile and use the root Kptfile to set the app name.
+
+<!--mdtogo-->
+
+You can learn more about recommended labels [here][recommended labels].
+
+<!--mdtogo:Long-->
+
+## Usage
+
+This function should be run in a KPT package. It does not require function config.
+
+### Run the function once
+```shell
+$ kpt fn eval --image set-default-labels:unstable
+```
+
+### Run the function in a Kpt pipeline
+
+```shell
+# add the function to Kptfile
```

Comment:
I'd suggest to move comment out of the code block:

Add the function to Kptfile:

```shell
$ kpt fn eval -t mutator -s -i set-default-labels:unstable
```

Reply:
yeah, I tested and realized that the bash comment will also be auto-copied by the copy-paste button.